integrations/webchat/agent-assist/agent-verify/README.md

```diff
@@ -0,0 +1,188 @@
+
+Note that the Test Simulator does NOT require an HTTP server. You can simply load the HTML file directly in the browser to test.
+
+## Genesys Setup
+There are two parts to the Genesys setup. The first part covers setting up the queue and creating the widget integration. This includes all the setup needed to embed the agent-verify widget into the Genesys agent dashboard. This README will walk you step by step through this setup. 
+
+The second part of the Genesys setup is to define the inbound call flow to reach the call queue that was set up in part 1 of this guide. The second part of the Genesys setup is not covered in this README. You will need to consult the Genesys documentation for details on how to setup the call flow. Note that there are several different options for this including:
+
+- the use of a Genesys provider to setup a phone number
+- the use of an external SIP trunking provider such as Twilio
+- the use of a Genesys call simulator
+
+### Step 1: Login to your Genesys account and navigate to the admin page
+
+![Genesys Admin Queue Page](images/admin-page-queue.jpg)
+
+From the Admin panel click on the `Queues` link to create a new queue that will be used to associate the agent-verify Interaction Widget with the agent dashboard that will be loaded whenever a call is answered off the queue.
+
+### Step 2: Create a new queue
+
+![Genesys Queue Page](images/create-queue-page.png)
+
+Click create queue and specify a name for the new queue. Read about queue options [here](https://help.mypurecloud.com/articles/create-queues).
+
+### Step 3: Configure the queue
+
+![Genesys Queue Page](images/queue-config-page.png)
+
+After specifying the members or groups associated with the new queue, you can simply take the defaults for the queue configuration with the exception of the `Voice Transcription` field outlined in red in the above image which needs to be set to On.
+
+
+### Step 4: Go to the Integrations page from the Admin panel
+
+![Genesys Admin Page](images/admin-page.png)
+
+From the Admin panel click on the `Integrations` link to create a new integration.
+
+### Step 5: Create a new integration
+
+![Genesys Integrations Page](images/integrations-page.png)
+
+Add a new integration by clicking the + sign in the top right of the page.
+
+### Step 6: Install the Interaction Widget
+
+![Genesys Interaction Widget Page](images/interaction-widget-page.png)
+
+The Interaction Widget is needed to embed third-party webapps via an iframe in the Agent UI.
+
+### Step 7: Configure the Interaction Widget
```

Comment:
I would reverse steps seven and eight. Because you need the client ID for the interaction widget, I would set that up first.